compiler/rustc_smir/src/rustc_smir/mod.rs

```diff
@@ -330,7 +330,16 @@ fn rustc_terminator_to_terminator(
             target: target.as_usize(),
             unwind: rustc_unwind_to_unwind(unwind),
         },
-        InlineAsm { .. } => todo!(),
+        InlineAsm { template, operands, options, line_spans, destination, unwind } => {
+            Terminator::InlineAsm {
+                template: format!("{:?}", template),
+                operands: format!("{:?}", operands),
```

Comment:
You should be able to easily convert the operand list, just like with the argument list of the call terminator

Reply:
Done!